perception/traffic_light_classifier/src/nodelet.cpp

```diff
@@ -98,19 +100,39 @@ void TrafficLightClassifierNodelet::imageRoiCallback(
   std::vector<cv::Mat> images;
   std::vector<size_t> backlight_indices;
   for (size_t i = 0; i < input_rois_msg->rois.size(); i++) {
-    output_msg.signals[i].traffic_light_id = input_rois_msg->rois.at(i).traffic_light_id;
+    // skip if the roi is not detected
```

Comment:
[must]
By using this approach, the order of traffic light output by detection/rois and classified/traffic_signals is different. 
As a result, the occlusion predictor seems to be miss-predict.

![image](https://github.com/autowarefoundation/autoware.universe/assets/37187849/2bcb5d5c-604b-4ed7-b971-370ba0a010dd)

Reply:
It seems necessary to either maintain the order with the classifier for traffic light or ensure ID matching with the occlusion predictor.

Reply:
Sorry for missing this point. I fixed it in https://github.com/autowarefoundation/autoware.universe/pull/5934/commits/981321eb7f087bcc7892afcecea15032b4d895be, by appending the undetected rois at the end of detection/rois, the order can be kept in the classifier.